src/hpr_routing.erl

```diff
@@ -113,10 +113,10 @@ packet_type(Packet) ->
         <<?JOIN_REQUEST:3, _:5, AppEUI:64/integer-unsigned-little,
             DevEUI:64/integer-unsigned-little, _DevNonce:2/binary, _MIC:4/binary>> ->
             {join_req, AppEUI, DevEUI};
-        (<<MType:3, _:5, DevAddr:32/integer-unsigned-little, _ADR:1, _ADRACKReq:1, _ACK:1, _RFU:1,
+        (<<FType:3, _:5, DevAddr:32/integer-unsigned-little, _ADR:1, _ADRACKReq:1, _ACK:1, _RFU:1,
```

Comment:
@mikev Which call(s) from https://github.com/helium/erlang-lorawan library should we call here instead of using Erlang's Binary syntax for descructuring-bind?

Reply:
Talked with @mikev we still need to have a function in there for that